engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendar.java

```diff
@@ -1214,14 +1216,16 @@ public LocalDate[] businessDates(final LocalDate start, final LocalDate end, fin
      * @throws InvalidDateException if the dates are not in the valid range
      * @throws DateTimeUtils.DateTimeParseException if the string cannot be parsed
      */
-    public LocalDate[] businessDates(final String start, final String end, final boolean startInclusive,
+    public String[] businessDates(final String start, final String end, final boolean startInclusive,
```

Comment:
Same comment as on the other overload.

Reply:
I assumed that String in implies String out.  If someone is working in strings, do you expect that they would not want this result in strings as well?

Reply:
I think `String in -> String out`, `LocalDate in -> LocalDate out` is the most user-friendly